perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -398,7 +400,7 @@ def Run(benchmark_spec):
 
   vm.PushFile(job_file_path, REMOTE_JOB_FILE_PATH)
 
-  if FLAGS.against_device:
+  if FLAGS.fio_target_mode in AGAINST_DEVICE_MODES:
```

Comment:
There are many repeat patterns like this: "FLAGS.fio_target_mode in AGAINST_DEVICE_MODES"
or "FLAGS.fio_target_mode not in AGAINST_DEVICE_MODES", consider making a private utility function that is simple one liner that returns a boolean for this.

e.g., 

def AgainstDevice() {
  return (blah)
}

Then in all these 'if' statements can be simplifed.